Resources/Prototypes/Entities/Structures/Piping/Atmospherics/pipes.yml

```diff
@@ -44,6 +48,7 @@
         offset: 0
       - !type:DoActsBehavior
         acts: ["Breakage"]
+
```

Comment:
why?

Reply:
what are you asking ?

Reply:
if its the linebreak that was def an accident